### ISSUE TITLE
propose alternative to openRegistration to indicate open registration with additional requirements

### ISSUE THREAD

**Issue by 6543** [jhass/nodeinfo#80]
as spam rises more instances do have individual registration prozedures like mail me at XY or ping me via ZY

**Comment by jhass** [comment]
I think `openRegistrations` can stay as is, maybe rewording its description slightly to emphasize self-service registration is possible.

It'll probably be hard to capture and keep up with all alternative methodology people come up with. Also frankly I doubt any statistical or survey value in knowing whether the registration scheme is say some individual creating accounts for friends and strangers or for example an  organizational instance that allows sign up via the organizations SSO. 

So my idea here would be to just have an optional field like `registrationRequirements` that should link to a document outlining the registration process for this instance. Aggregators can then show registrations as for example "Open" / "Closed" / "Restricted" based on the combinations of `openRegistration` and the presence of that new field.

What do you think?

**Comment by 6543** [comment]
I was also going tobpropose something that just contain link(s) as we shoud not statically definebsomething as there are to many options ;)

=> i agree

As for `registrationRequirements` idea, i would say add to `openRegistration` that it should be true, if optional registrationReq. are set.
Or that parsers should ignore it if regis.Req. are set
